src/main/java/io/github/rabobank/shadow_tool/EncryptionService.java

```diff
@@ -3,22 +3,16 @@
 import org.bouncycastle.util.encoders.Base64;
 
 import javax.crypto.Cipher;
-import java.security.PublicKey;
-
 
 class EncryptionService {
-    public static final String ALGORITHM = "RSA";
-    public static final String ALGORITHM_MODE_PADDING = ALGORITHM + "/ECB/OAEPWITHSHA-256ANDMGF1PADDING";
-    private final PublicKey publicKey;
+    private final Cipher cipher;
 
-    EncryptionService(final PublicKey publicKey) {
-        this.publicKey = publicKey;
+    EncryptionService(final Cipher cipher) {
+        this.cipher = cipher;
     }
 
-    String encrypt(final String value) {
+    public String encrypt(final String value) {
```

Comment:
I don't see a need for this to be public, package private should be fine.

Reply:
My bad, not sure when and why this happened